bin/check-uptime.rb

```diff
@@ -31,16 +32,33 @@
 class CheckUptime < Sensu::Plugin::Check::CLI
   option :warn,
          short: '-w SEC ',
-         description: 'Warn if uptime is below SEC',
+         description: 'Warn threshold in SEC',
          proc: proc(&:to_i),
          default: 180
 
+  option :greather,
+         short: '-g',
+         long: '--greater-than',
+         description: 'This compare uptime > threshold (seconds)',
+         boolean: true,
+         default: false
+
+  option :less,
```

Comment:
What's the value in having both of these options exposed? If we are gonna do that I would opt for just passing in the operator rather than 2 booleans that are mutually exclusive. I'd vote for 1 option (boolean or string param) with an obvious name.